bbn-test-4/finality-providers/registry/taro.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": {
+    "moniker": "taro",
+    "identity": "",
+    "website": "",
+    "security_contact": "",
```

Comment:
Thanks! I would appreciate it if you could specify a contact and recreate the signature.

Reply:
I've added my contact information, hope it's not too late!